app/styles/_membership.less

```diff
@@ -31,7 +31,7 @@
     .col-name {
       min-width: 165px;
       padding: 0 5px 10px 0;
-      word-wrap: break-word;
+      word-break: break-all;
```

Comment:
is there a reason you aren't using our word-break (i think is the name) mixin? it makes sure the best option is used for each browser